php_phongo.c

```diff
@@ -916,15 +892,33 @@ int phongo_execute_command(mongoc_client_t *client, php_phongo_command_type_t ty
 		bson_destroy(&reply);
 	}
 
-	if (!phongo_advance_cursor_and_check_for_error(cmd_cursor TSRMLS_CC)) {
-		/* If an error is found, the cmd_cursor is destroyed already */
-		return false;
-	}
-
 	phongo_cursor_init_for_command(return_value, client, cmd_cursor, db, zcommand, zreadPreference TSRMLS_CC);
 	return true;
 } /* }}} */
 
+/* Advance the cursor and return whether there is an error. On error, false is
```

Comment:
Moving this down here made it impossible to see the changes in the diff, can you please put it back in its original location? That way, I would have spotted that you removed the ``mongoc_cursor_destroy(cursor);`` from it.